webpack.config.common.js

```diff
@@ -50,40 +28,73 @@ const sharedConfig = {
         }),
         new MiniCssExtractPlugin({ filename: 'molstar.css' }),
         new VersionFilePlugin(),
-    ],
-    resolve: {
-        modules: [
-            'node_modules',
-            path.resolve(__dirname, 'lib/')
-        ],
-        fallback: {
-            fs: false,
-            vm: false,
-            buffer: false,
-            crypto: require.resolve('crypto-browserify'),
-            path: require.resolve('path-browserify'),
-            stream: require.resolve('stream-browserify'),
-        }
-    },
-    watchOptions: {
-        aggregateTimeout: 750
+    ];
+
+    if (copyPluginPatterns && copyPluginPatterns.length > 0) {
+        plugins.push(new CopyPlugin({
+            patterns: copyPluginPatterns,
+        }));
     }
-};
+
+    return {
+        module: {
+            rules: [
+                {
+                    test: /\.(html|ico|webmanifest|png)$/,
+                    use: [{
+                        loader: 'file-loader',
+                        options: { name: '[name].[ext]' }
+                    }]
+                },
+                {
+                    test: /\.(s*)css$/,
+                    use: [
+                        MiniCssExtractPlugin.loader,
+                        { loader: 'css-loader', options: { sourceMap: false } },
+                        { loader: 'sass-loader', options: { sourceMap: false } },
+                    ]
+                },
+                {
+                    test: /\.(jpg)$/i,
+                    type: 'asset/resource',
+                },
+            ]
+        },
+        plugins: plugins,
+        resolve: {
+            modules: [
+                'node_modules',
+                path.resolve(__dirname, 'lib/')
+            ],
+            fallback: {
+                fs: false,
+                vm: false,
+                buffer: false,
+                crypto: require.resolve('crypto-browserify'),
+                path: require.resolve('path-browserify'),
+                stream: require.resolve('stream-browserify'),
+            }
+        },
+        watchOptions: {
+            aggregateTimeout: 750
+        }
+    };
+}
 
 function createEntry(src, outFolder, outFilename, isNode) {
     return {
         target: isNode ? 'node' : void 0,
         entry: path.resolve(__dirname, `lib/${src}.js`),
         output: { filename: `${outFilename}.js`, path: path.resolve(__dirname, `build/${outFolder}`) },
-        ...sharedConfig
+        ...getSharedConfig()
     };
 }
 
-function createEntryPoint(name, dir, out, library) {
+function createEntryPoint(name, dir, out, library, copyPluginPatterns) {
```

Comment:
```suggestion
function createEntryPoint(name, dir, out, library, options = {}) {
```

And use `options.copyPluginPatters`